### PR TITLE
bump quickcheck to 0.6

### PR DESCRIPTION
**Pull request by igor-raits**

### CODE REVIEW

File: Cargo.toml [hunk 1]
```diff
@@ -12,4 +12,4 @@ keywords = ["text-processing", "search", "spelling"]
 license = "Apache-2.0"
 
 [dev-dependencies]
-quickcheck = "^0.4.1"
+quickcheck = "0.6"
```
Comment (febeling) [inline Cargo.toml:15]: @ignatenkobrain - Is there a specific reason you switched to fixed version spec?

Comment (igor-raits) [reply Cargo.toml:15]: @febeling it's not fixed version. Not specifying OP means `^`. so `0.6` == `^0.6`

Comment (febeling) [reply Cargo.toml:15]: @ignatenkobrain cool, sounds good!

**Comment by febeling** [comment]
Thanks for the PR, @ignatenkobrain!

**Pull request merged**